react-payments-app/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,22 @@
+const Header = ({ pageTitle, headerIcon, onClick }) => {
+  return (
+    <>
+      <div id="header" className="page-title mb-10">
+        {headerIcon && (
+          <div
+            id="icon"
+            className="cursor-pointer"
+            onClick={(e) => {
+              onClick(e);
+            }}
```

Comment:
이렇게 쓰신 이유가 있으실까요? 

```suggestion
            onClick={onClick}
```

위와 같이 써도 충분할거 같아서요 😄